core/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+import uuid
+from accounts.models import User
+# Create your models here.
+class Collections(models.Model):
+    id = models.UUIDField(default=uuid.uuid4,primary_key=True)
+    user = models.ForeignKey(User,on_delete=models.CASCADE)
+    name = models.CharField(max_length=50)
+
+
+class HandwritingInputLogger(models.Model):
+    id = models.UUIDField(default=uuid.uuid4,primary_key=True)
+    collection = models.ForeignKey(Collections,on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+    input_file = models.FileField(upload_to='inputFiles/')
+    status = models.BooleanField(default=False)
```

Comment:
What is this status for?  error status  is also there you can merge both

Reply:
Since we are doing it in the background. There is a possibility that notification is missed. So, we thought app will be requesting every 3 seconds to check the status till the notification is delivered.